modules/packages/NetCDF.chpl

```diff
@@ -17,7 +17,29 @@
  * limitations under the License.
  */
 
+/* NetCDF bindings for Chapel
+
+This module implements the C-API for netCDF version 4.6.1.  It is expected
```

Comment:
I tend to prefer documenting things for what they are, and leaving planned features / minor bugs in issues, so I'd lean towards leaving this out. That said, I don't think there is an established practice on this, so I'll leave it up your judgement.